datacatalog/cloud-client/lookup_entry_test.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from gcp_devrel.testing import eventually_consistent
+
+import lookup_entry
+
+PROJECT = os.environ['GCLOUD_PROJECT']
+BIGQUERY_DATASET = os.environ['GCLOUD_BIGQUERY_DATASET']
```

Comment:
Are these environment variables set in our test config already? I guess they might be if these are the same that are used in BigQuery / Pub/Sub tests?

Reply:
Only GCLOUD_PROJECT is set in https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/testing/test-env.tmpl.sh. There is PUBSUB_TOPIC env var there, but for a different purpose. Data Catalog tests need existing BigQuery and PubSub assets to succeed. In this case, maybe it will be better to create/delete them using a fixture to run such tests. Agree?

Reply:
Either create in a fixture or use a public dataset. I recommend a public dataset for BigQuery. I believe there's one (taxi data) for Pub/Sub, too.

Reply:
Done! Tks.